browser/brave_local_state_prefs.cc

```diff
@@ -90,18 +89,15 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
       metrics::prefs::kMetricsReportingEnabled,
       base::Value(GetDefaultPrefValueForMetricsReporting()));
 
-#if BUILDFLAG(BRAVE_P3A_ENABLED)
-  brave::BraveP3AService::RegisterPrefs(registry,
+  p3a::P3AService::RegisterPrefs(registry,
 #if !BUILDFLAG(IS_ANDROID)
-                                        first_run::IsChromeFirstRun());
+                                 first_run::IsChromeFirstRun());
```

Comment:
what is going on with the indentation here?

Reply:
this is what `npm run presubmit -- --fix` did for me

Reply:
oic, these are params to `RegisterPrefs` so I guess the indentation is ok now that it removed the tab that was in there